### ISSUE TITLE
4.x tutorial pages have 3.x in the page title

### ISSUE THREAD

**Issue by kcoleman-marklogic** [marklogic/marklogic-data-hub#1364]
We missed a step somewhere in prepping the documentation for publication in 4.0.0. The banner across the top of every page of the tutorial says "Getting Started Tutorial 3.x". The content is correct, only the page title is wrong.

We also neglected to create a link to the real 3.x version of the tutorial on the first page of the current tutorial. There are only links to 1.x and 2.x. Do we even have the 3.x tutorial available somewhere right now?

**Comment by kcoleman-marklogic** [comment]
This is going to be a mess to unwind, I think. It seems that I should have modified the tutorial for 4.0.0 by first making a copy of the 3.x tutorial in a _pages/tutorial/4x folder. That should have occurred to me. Mea culpa.

The difficulty now will be recreating the 3.x page set.

**Comment by grechaw** [comment]
I'm going to push for no inclusion of 3.x tutorial.  No reason to get people started on an older release.  Fortunately, were we to recreate the old ones its just a little git machination.

**Comment by grechaw** [comment]
Agreed, we must fix this right away

**Comment by kcoleman-marklogic** [comment]
Right away = after the current release kerfluffle, assuming I don't have to shuffle everything around in order to fix the cosmetic aspect of this. Might still be post 4.0.1, even so. 

I have a sinking feeling there's some over-engineered jekyll magic in play that keys off the folder name, but I haven't gone to look yet. Jello, as you say.

**Comment by grechaw** [comment]
i meant we there not you :)    You've plenty to wrangle with in the short term.

**Comment by kcoleman-marklogic** [comment]
So pull req #1364 will make everything look tidy on the surface, but here's some things to keep in mind when dealing with the structural issue:

* There is no jekyll magic at play here, which is good.
* If we change the folder name, all kinds of links to tutorial pages need to be updated because, God help us, all the images get generated under _site/images/3x/...
* There's a hub_version variable lazing about in _data/global.yml, but it doesn't seem to be used. Nor could I figure out how to get it to work. I'm sure there's a way (though not in the page titles, from what I've read).

Bottom line is that the tutorial is not structurally set up for long term maintenance. Making a copy of it and fixing up all the links and other references every time there's a new major version is not how we support multiple versions, kids.  This ties back to DHFPROD-912.

**Comment by grechaw** [comment]
OK, glad it ties back to an issue inside, Ill close this out as fixed, sorta
